src/main/java/microservices/userServiceBasic/domain/model/User.java

```diff
@@ -0,0 +1,64 @@
+package microservices.userServiceBasic.domain.model;
+
+import java.util.UUID;
+
+public class User {
+
+	private UUID id;
+	private String name;
+	private String lastName;
+	private String email;
+	private String address;
+	private String nationality;
+
+	public User(String name, String lastName, String email) {
+		this.id = UUID.randomUUID();
```

Comment:
A POJO/model should not have any logic. Generating the UUID should be generated elsewhere. You can put it in the UserService for now.